ROADMAP.md

```diff
@@ -1 +1,39 @@
+# Road Map
 
+## What
+We want to build an application/tool that can import a dataset, store the relevant information in a database, then dispay the data to the user. We will accomplish this in 3 milestones:  
```

Comment:
I'd be more specific about the type of application/tool we want to build. Web application perhaps?

Reply:
I have specified this as a web application